dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -367,8 +409,8 @@ def _collect_aggregations(
             # Gather information about the input column
             inputs = agg.getArgs(expr)
 
-            # TODO: This if statement is likely no longer needed but left here for the time being just in case
-            if aggregation_name == "regr_count":
+            # if aggregation_name in ["regr_count", "regr_syy", "regr_sxx"]:
+            if len(inputs) == 2:
```

Comment:
Looks like this code was initially designed where `regr_count` is the only aggregation we expected to have 2 inputs. I changed it to `if len(inputs) == 2` instead, although I'm not sure that's what we want with the null logic used for `regr_count` that is included in the code block below.

Maybe `if aggregation_name in ["regr_count", "regr_syy", "regr_sxx"]` would be better?